### PR TITLE
Add some styles to RWD and new breakpoints

### PR DESCRIPTION
**Pull request by MiroslawChmielecki**
I added some styles to section PROMOTION and now is RWD, and also I created new breakpoints which I needed.  I corrected existing breakpoints according to bootstrap, hope I didn't break anything :)

### CODE REVIEW

File: src/sass/components/_section--promotion.scss [hunk 1]
```diff
@@ -6,6 +6,9 @@
     background-image: url("../images/whiteSofa.jpg");
     background-size: cover;
     background-position: center;
+    @include small {
```
Comment (matwjablonski) [inline src/sass/components/_section--promotion.scss]: add empty line before

File: src/sass/components/_section--promotion.scss [hunk 2]
```diff
@@ -47,6 +50,9 @@
     background-color: $promotion-primary-color;
     height: 195px;
     margin-bottom: 30px;
+    @include small {
```
Comment (matwjablonski) [inline src/sass/components/_section--promotion.scss]: add empty line before

File: src/sass/components/_section--promotion.scss [hunk 3]
```diff
@@ -63,6 +69,9 @@
       padding: 0 20px;
       min-width: 120px;
       text-transform: uppercase;
+      @include large {
```
Comment (matwjablonski) [inline src/sass/components/_section--promotion.scss]: add empty line before

File: src/sass/components/_section--promotion.scss [hunk 4]
```diff
@@ -116,6 +125,9 @@
       flex-direction: column;
       align-items: flex-end;
       text-transform: uppercase;
+      @include large {
```
Comment (matwjablonski) [inline src/sass/components/_section--promotion.scss]: add empty line before

File: src/sass/mixins/_breakpoints.scss [hunk 5]
```diff
@@ -1,17 +1,41 @@
+//max-width: 575px
 @mixin phones {
-  @media (max-width: $bp-s) {
+  @media (max-width: $bp-s - 1) {
```
Comment (matwjablonski) [inline src/sass/mixins/_breakpoints.scss]: are you sure that your changes in breakpoint doesn't affect on other parts of projects?

File: src/sass/components/_section--promotion.scss (hunk 1 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: fixed !!

File: src/sass/components/_section--promotion.scss (hunk 2 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: fixed !!

File: src/sass/components/_section--promotion.scss (hunk 3 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: fixed !!

File: src/sass/components/_section--promotion.scss (hunk 4 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: fixed !!

File: src/sass/mixins/_breakpoints.scss (hunk 5 above)
Comment (MiroslawChmielecki) [reply src/sass/mixins/_breakpoints.scss]: I'm not sure so I deleted my changes in other's breakpoints and I left only my additional breakpoints which I used in code

**Pull request merged**